src/screens/NewAnalytics/PaymentAnalytics/PaymentsLifeCycle/PaymentsLifeCycle.res

```diff
@@ -0,0 +1,29 @@
+open NewAnalyticsTypes
+open NewAnalyticsHelper
+open SankeyGraphTypes
+@react.component
+let make = (
+  ~entity: moduleEntity,
+  ~chartEntity: chartEntity<sankeyPayload, sankeyGraphOptions>,
+) => {
+  let (paymentsLifeCycle, setPaymentsLifeCycle) = React.useState(_ => JSON.Encode.null)
+  let getPaymentLieCycleData = async () => {
```

Comment:
why async? only set State is happening

Reply:
will add a api call here once the api is ready, now its just loading hard coded values